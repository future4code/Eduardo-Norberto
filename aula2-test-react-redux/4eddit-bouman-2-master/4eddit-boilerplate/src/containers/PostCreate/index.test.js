import React from "react"
import renderer from "react-test-renderer"
import { Create, CreatePostContainer } from "./index"
import { shallow } from "enzyme"


describe("Testa o componente PostCreate", ()=> {
    test('Snapshot', ()=>{
        
        const createPost = renderer.create(<Create />).toJSON();
        expect(createPost).toMatchSnapshot();
    })
    
})