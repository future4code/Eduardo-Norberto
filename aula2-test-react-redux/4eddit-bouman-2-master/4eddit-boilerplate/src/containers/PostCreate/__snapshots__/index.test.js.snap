// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testa o componente PostCreate Snapshot 1`] = `
<form
  className="sc-fzXfMD jodrTC"
/>
`;
